lib/fortnox/api/types.rb

```diff
@@ -25,7 +21,7 @@ def is?(option_name)
 module Fortnox
   module API
     module Types
-      include Dry::Types.module
+      include Dry.Types(default: :nominal)
```

Comment:
Have you checked the different default options here?

Reply:
`:nominal` is backwards compatible. 

https://github.com/dry-rb/dry-types/blob/main/lib/dry/types.rb#L69